src/hooks/sponsorships.ts

```diff
@@ -198,33 +199,7 @@ export function invalidateSponsorshipByIdQueries(sponsorshipId: string) {
 export function useSponsorshipByIdQuery(sponsorshipId: string) {
     return useQuery({
         queryKey: ['useSponsorshipByIdQuery', sponsorshipId],
-        async queryFn() {
-            let rawSponsorship: Sponsorship | undefined | null
-
-            try {
-                rawSponsorship = (await getSponsorshipById(sponsorshipId, {
-                    force: true,
-                })) as Sponsorship | null
-            } catch (e) {
-                console.warn('Failed to fetch a Sponsorship', e)
-
-                errorToast({ title: 'Could not fetch Sponsorship details' })
-            }
-
-            if (!rawSponsorship) {
-                return null
-            }
-
-            try {
-                return SponsorshipParser.parseAsync(rawSponsorship)
-            } catch (e) {
-                console.warn('Failed to parse a Sponsorship', e)
-
-                errorToast({ title: 'Could not parse Sponsorship details' })
-            }
-
-            return null
-        },
+        queryFn: () => getParsedSponsorshipById(sponsorshipId, { force: true }),
```

Comment:
The replaced code here is literally the new `getParsedSponsorshipById` = rearrangement.